### ISSUE TITLE
Example playbook fails in Vagrant

### ISSUE THREAD

**Issue by clebio** [karlmdavis/ansible-jenkins2#12]
The `Example Playbook` in the README doesn't work out of the box.

Using for instance:
```
- hosts: all
  vars:
    jenkins_port: 8080

  roles:
     - karlmdavis.jenkins2
```

In Vagrant, gives:
```
TASK [karlmdavis.jenkins2 : Update APT Cache] **********************************
fatal: [jenkins]: FAILED! => {"changed": false, "failed": true, "msg": "Failed to lock apt for exclusive operation"}
        to retry, use: --limit @/Users/caleb/dev/ansible-galaxies/jenkins.retry
```

This is simple enough to fix by adding `become: yes` here or there.

**Comment by karlmdavis** [comment]
Interesting that the tests didn't catch this! I guess the APT cache is always already updated when they're running.

**Comment by karlmdavis** [comment]
Resolved in PR #13.

**Comment by karlmdavis** [comment]
Thanks for the report!
